crates/bevy_color/Cargo.toml

```
@@ -17,7 +17,7 @@ bevy_reflect = { path = "../bevy_reflect", version = "0.14.0-dev", features = [
 bytemuck = { version = "1", features = ["derive"] }
 serde = { version = "1.0", features = ["derive"], optional = true }
 thiserror = "1.0"
-wgpu-types = { version = "0.19", default-features = false, optional = true }
+wgpu-types = { version = "0.20", default-features = false, optional = true }
```

```suggestion
wgpu-types = { version = "0.20.1", default-features = false, optional = true }
```

If 0.20.0 was so problematic, we should prevent it from being pulled in by making 0.20.1 the minimum version.